memory/vatest.py

```diff
@@ -68,11 +79,14 @@ def setUp(self):
             for hp_size in self.hsizes:
                 nr_pgs = int((total_mem / 2) / hp_size)
                 genio.write_file(
-                    '/sys/kernel/mm/hugepages/hugepages-%skB/nr_hugepages' % str(hp_size * 1024), str(nr_pgs))
+                    '/sys/kernel/mm/hugepages/hugepages-%skB/'
+                    'nr_hugepages' % str(hp_size * 1024), str(nr_pgs))
             n_pages = genio.read_file(
-                '/sys/kernel/mm/hugepages/hugepages-%skB/nr_hugepages' % str(self.hsizes[0] * 1024)).rstrip("\n")
+                '/sys/kernel/mm/hugepages/hugepages-%skB/nr_huge'
```

Comment:
`/sys/kernel/mm/hugepages/hugepages-%skB/nr_hugepages` is used in many of the places ... where ever possible Create an variable and use the same.

Reply:
Sure, makes sense